package.json

```diff
@@ -10,7 +10,8 @@
         "lint:tsc": "tsc --noEmit",
         "postinstall": "husky install",
         "test": "jest",
-        "test:watch": "jest --watch"
+        "test:watch": "jest --watch",
+        "start": "node dist/index.js $ORIGIN_URL --port $PORT --cacheSizeLimitHint $CACHE_SIZE_LIMIT_HINT --cacheDir $CACHE_DIR"
```

Comment:
hm, what if an env var is not set (or a new one is added?)

alternatively you could pack them all into one xxx_OPTS env (except ORIGIN_URL)